### ISSUE TITLE
Error prisma.yml doesn't exist in

### ISSUE THREAD

**Issue by FrancescoSaverioZuppichini** [prisma/upgrade#154]
I am updating from prisma 1 to 2 but I got this error

```
 npx prisma-upgrade 
Error prisma.yml doesn't exist in "/Users/francescozuppichini/Documents/taxonomy-main/prisma/prisma.yml". Run `prisma-upgrade -h` for more details.
```

Thanks a lot

**Comment by janpio** [comment]
Does `/Users/francescozuppichini/Documents/taxonomy-main/prisma/prisma.yml` exist?

**Comment by FrancescoSaverioZuppichini** [comment]
> Does `/Users/francescozuppichini/Documents/taxonomy-main/prisma/prisma.yml` exist?

No, but maybe I am already using prisma v2 hahah - is there a way I can check if the layer is the rust one? I received an error in vscode saying I might still using the graphql one

**Comment by janpio** [comment]
What version of `prisma`, `@prisma/cli` or `@prisma/client` are you using in your `package.json` of your project?
